mapswipe_workers/mapswipe_workers/generate_stats/generate_stats.py

```diff
@@ -0,0 +1,46 @@
+from mapswipe_workers.definitions import logger
+from mapswipe_workers.definitions import DATA_PATH
+
+from mapswipe_workers.generate_stats import project_stats
+from mapswipe_workers.generate_stats import overall_stats
+
+
+def generate_stats(project_id_list, all_projects=False):
```

Comment:
I would add a docstring to this function, to make it clear what it does. The one in `transfer_results` was pretty good.